superset/assets/visualizations/paired_ttest.jsx

```diff
@@ -0,0 +1,266 @@
+import d3 from 'd3';
+import dist from 'distributions';
+
+import React from 'react';
+import { Table, Tr, Td, Thead, Th } from 'reactable';
+import ReactDOM from 'react-dom';
+import PropTypes from 'prop-types';
+
+import './paired_ttest.css';
+
+class TTestTable extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      pValues: [],
+      liftValues: [],
+      control: 0,
+    };
+  }
+
+  componentWillMount() {
+    this.computeTTest(this.state.control);
+  }
+
+  getLiftStatus(row) {
```

Comment:
Could you add some code comments on some of these functions? I get the general idea of what the functions do, but some comments might help with future clarity.